### ISSUE TITLE
prediction mask is black 

### ISSUE THREAD

**Issue by yomamdde** [asujaykk/Road-segmentation-UNET-model#1]
/Road-segmentation-UNET-model/data/data_set/data_temp_folder/

I trained my model without data augmentation, using the images in the '/Road-segmentation-UNET -model/data/data_set/data_temp_folder/' .
After the training was completed, an h5 file was generated. 
However, when I tested predictions using the images in the '/Road-segmentation-UNET-model/data/data_set/data_temp_folder/' folder, I only got black screens as the output. 
when I tested your pre-trained road_segmentation_160_160.h5 model, the results were quite good.

road_segmentation_160_160.h5 (you)
![image](https://github.com/asujaykk/Road-segmentation-UNET-model/assets/108274814/b73230a7-3727-4aa9-93c0-792f9d1f1796)

road_segmentation_160_160_test.h5 (me)
![image](https://github.com/asujaykk/Road-segmentation-UNET-model/assets/108274814/1cadb41e-687d-41fc-9113-096bb26035c7)

**Comment by asujaykk** [comment]
Hello @yomamdde ,
This project was intended  to demonstrate the road segmentation model with U-net Architecture. And the data was completely  annotated by me and they were very less.
 The model won't be able to learn much from the limited number of images, that's why I recommend to use augmentation. The  model which I shared here was tained on the augmented data set (just for demo purpose).

Additional hint: 
Do not expect this pretrained model work on your data ,because the data set on which the model  trained  was very less.

 Instead you can retrain the model with your on data for better output. The training method and expected input format are explained in the repo. Please try it.
I appreciate if you can provide the outcome after trying it on your data. 😊
